packages/backend/src/app.ts

```diff
@@ -253,6 +256,24 @@ export class App {
     this.koa.use(this.publicRouter.middleware())
   }
 
+  private async processAccount(): Promise<void> {
```

Comment:
There are enough of these jobs that it might be worth abstracting the job processing a bit so that these `processFoo` functions are less repetitive. (Doesn't need to be in this PR though).